tests/unit/list/PageableList.js

```diff
@@ -1467,7 +1467,7 @@ define([
 								assertCategorizedList(list, 50, 25, true, true);
 								assert.strictEqual(
 									removeTabsAndReturns(list._getLastVisibleRenderer().textContent),
-									"item 73",
+									"Category 5", //"item 73",
```

Comment:
Looks like you just modified the test to hide the failure?

Reply:
Same as PageabkeList-ObservableArray.js